packages/xo-server/src/api/host.js

```diff
@@ -314,3 +314,23 @@ scanPifs.params = {
 scanPifs.resolve = {
   host: ['id', 'host', 'administrate'],
 }
+
+// -------------------------------------------------------------------
+
+export async function installCertificate({ host, ...props }) {
+  host = this.getXapiObject(host)
+  await host.$xapi.installCertificateOnHost(host.$id, props)
```

Comment:
`$id` is an XO identifier, which may be unrelated to the XAPI ref or UUID.

```suggestion
  await host.$xapi.installCertificateOnHost(host._xapiId, props)
```

Reply:
I think it's the same thing. See https://github.com/vatesfr/xen-orchestra/tree/master/packages/xen-api

Reply:
That's only true for XAPI objects, not XO objects.

Reply:
But it's an XAPI object.

Reply:
My bad :sweat: 

It's perfect as it is :slightly_smiling_face: